examples/plot_clip.py

```diff
@@ -0,0 +1,102 @@
+"""
+Clip Vector Data with EarthPy
+==================================================================
+
+Learn how to clip point, line, or polygon geometries with a
+polygon geometry using EarthPy.
+
+"""
+
+###############################################################################
+# Clip Vector Data in Python Using EarthPy
+# ---------------------------------------------
+#
+# The example below walks you through a typical workflow for clipping one
+# vector data file to the shape of another. Both vector data files must be
+# opened with GeoPandas as GeoDataFrames and be in the same Coordinate
+# Reference System (CRS) for the ``clip_shp()`` function in EarthPy to work.
+# The object to be clipped will be clipped to the full extent of the clip
```

Comment:
I suggest adding a sentence about the data here at the end of this paragraph (after line 17), similar to what you have on lines 43-44. Something like: "This example uses a line shapefile containing all major roads in North America and a polygon shapefile containing the United States boundary to show you how to prepare and clip vector data. In this example, the roads will be clipped to the United States boundary."